packages/web3wallet/src/controllers/engine.ts

```diff
@@ -47,11 +47,11 @@ export class Engine extends IWeb3WalletEngine {
   };
 
   public updateSession: IWeb3WalletEngine["updateSession"] = async (params) => {
-    return await (await this.signClient.update(params)).acknowledged();
+    return await this.signClient.update(params);
   };
 
   public extendSession: IWeb3WalletEngine["extendSession"] = async (params) => {
-    return await (await this.signClient.extend(params)).acknowledged();
```

Comment:
While obviously nicer in terms of performance, doesn't this weaken guarantees about clients' sessions being in sync? E.g. clientA (calling) ends up with an extended session for it then to expire unexpectedly on clientB (responding) bc it didn't explicitly handle the extension.

Reply:
>doesn't this weaken guarantees about clients' sessions being in sync

Kind of, in cases when the dapp is offline BUT It has the big benefit of actually publishing update to all sessions regardless if the dapp is online. This solves an issue like 

- wallet has two session that wants to update
- one with active dapp (thats online), one without
- wallet tries to update both
- session with online dapp succeeds, session with offline dapp is stuck

With users having tons of active sessions on their wallet, chances are not all of them most will be offline. 
This just clogs the SDK with tasks that will never resolve.